src/components/FilterCollapse.js

```diff
@@ -126,9 +136,12 @@ const FilterCollapse = ({ isOpen, hideDescription }) => {
               placeholder={'name:"Ambush Viper"'}
               valid={filterInput.length > 0 && filterValid}
               invalid={filterInput.length > 0 && !filterValid}
-              value={filterInput}
-              onChange={(event) => setFilterInput(event.target.value)}
+              value={searchFilterInput}
+              onChange={(event) => setSearchFilterInput(event.target.value)}
             />
+            <Button color="success" onClick={applySearchFilter}>
+              Apply Filter
```

Comment:
As a historical note, this button used to just say "Apply" before it was removed. I don't known whether this is better, but the advanced filter modal also uses just "Apply", so the shorter variant at least scores a point for consistency.